sklearn/linear_model/stochastic_gradient.py

```diff
@@ -143,45 +140,9 @@ def _validate_params(self, set_max_iter=True, for_partial_fit=False):
 
         if not set_max_iter:
             return
-        # n_iter deprecation, set self._max_iter, self._tol
+
         self._tol = self.tol
```

Comment:
I'm not sure we need these private attributes anymore. I think they were invented for the deprecation

Reply:
right, I removed them